infra/helper.py

```diff
@@ -205,6 +224,8 @@ def build_fuzzers(build_args):
 def run_fuzzer(run_args):
   """Runs a fuzzer in the container."""
   parser = argparse.ArgumentParser('helper.py run_fuzzer')
+  _add_engine_args(parser)
```

Comment:
It should be able to detect engine based on the build in /out, lets make it simple by not needing to explicitly indicate it.